invenio_rdm_records/records/jsonschemas/records/parent-v3.0.0.json

```diff
@@ -0,0 +1,93 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "id": "local://records/parent-v3.0.0.json",
+  "title": "InvenioRDM Parent Record Schema v3.0.0",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "$schema": {
+      "description": "JSONSchema declaration.",
+      "type": "string"
+    },
+
+    "id": {
+      "description": "Persistent record identifier (alphanumeric).",
+      "type": "string"
+    },
+
+    "pid": {"$ref": "local://records/definitions-v1.2.0.json#/internal-pid"},
+
+    "access": {
+      "type": "object",
+      "description": "Access control and ownership for all versions of a record.",
+      "additionalProperties": false,
+      "properties": {
+
+        "owned_by": {
+          "description": "Owner of the child records.",
+          "$ref": "local://records/definitions-v2.0.0.json#/agent"
+        },
+
+        "grants": {
+          "description": "Access grants for the child records.",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "required": ["subject", "permission"],
+            "additionalProperties": false,
+            "properties": {
+              "subject": {
+                "type": "object",
+                "required": ["type", "id"],
+                "additionalProperties": false,
+                "properties": {
+                  "type": {
+                    "type": "string"
```

Comment:
removed the `enum` here, because we agreed that the content validation should be done by marshmallow rather than JSON Schema